### PR TITLE
More details in package.json

### PR DESCRIPTION
**Pull request by cbeauchesne**

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -1,7 +1,18 @@
 {
-  "name": "vue-camptocamp",
-  "version": "0.1.0",
-  "private": true,
+  "name": "c2c_ui",
+  "description": "https://www.camptocamp.org user interface",
+  "version": "7.0.0beta",
```
Comment (brunobesson) [inline package.json]: Do we really need to start from 7.0?

Comment (cbeauchesne) [reply package.json]: We start from whatever we want. Increasing major version seems to be obvious, but that's a number that may cause some reaction. I've hesitate to skip 7, and go directly to 8 for this reason :) 

Whatever, what is your opinion on this ?

**Comment by brunobesson** [comment]
1.0.0?

**Comment by cbeauchesne** [comment]
As this project is a refactoring from another project, don't you think it's kinda weird to start from 1.0, which means a totally new project ?

@asaunier, you own the legacy of c2c projects, what is your opinion ?

**Comment by asaunier** [comment]
@cbeauchesne I own nothing :P 

Since c2c_ui fully replaces the current UI whose latest release is v6.2.21, it makes sense to refer to it as UI v7.x
What version number will be used on the forums? I'm pretty sure every one will be confused if the new UI is refered to as "v1".

**Comment by cbeauchesne** [comment]
@asaunier 

> What version number will be used when discussing about the new UI on the forums? 

"ui-2019" (a tag on forum), "la maquette", "la nouvelle interface"... Nothing that looks like a version number.

> I'm pretty sure every one will be confused if the new UI is refered to as "v1".

I agree.

**Comment by brunobesson** [comment]
Let it be 7.0.0 then

**Comment by cbeauchesne** [comment]
O, let's go! thanks.

**Pull request merged**